.github/workflows/builder_maven_slsa3.yml

```diff
@@ -27,6 +27,11 @@ on:
         required: false
         default: 17
         type: number
+      directory:
+        description: "Sub-directory to launch the build from. Must be under the workspace. Relative from the root of the file directory when invoking the builder. Should not include a '/' at the end."
```

Comment:
by taking the `realpath` of this path, it should eliminate the trailing `/`. This way users need not have this additional  constraints.

Reply:
Yes, let's do that.